src/main/java/com/onarandombox/MultiverseCore/commands/GameruleCommand.java

```diff
@@ -64,17 +65,38 @@ public void runCommand(CommandSender sender, List<String> args) {
             world = Bukkit.getWorld(args.get(2));
             if (world == null) {
                 sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " World " + ChatColor.AQUA + args.get(2)
-                    + ChatColor.WHITE + " does not exist.");
+                        + ChatColor.WHITE + " does not exist.");
                 return;
             }
         }
 
-        if (world.setGameRuleValue(gameRule, value)) {
-            sender.sendMessage(ChatColor.GREEN + "Success!" + ChatColor.WHITE + " Gamerule " + ChatColor.AQUA + gameRule
-                    + ChatColor.WHITE + " was set to " + ChatColor.GREEN + value);
+        if (gameRule == null) {
+            sender.sendMessage(ChatColor.RED + "Failure! " + ChatColor.AQUA + args.get(0) + ChatColor.WHITE
+                    + " is not a valid gamerule.");
         } else {
-            sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " Gamerule " + ChatColor.AQUA + gameRule
-                    + ChatColor.WHITE + " cannot be set to " + ChatColor.RED + value);
+            boolean success = false;
```

Comment:
I'm not keen on this kind of flag setting pattern for something like this. Usually the following can be written in such a way that you don't need to track a flag like this through the code. Rationale: It can be easy to lose track of what happened to this flag by the end of the following code block when the flag is read and acted on. It would be more clear what was happening if the success condition was obvious when it was needed to be known. I can go into more detail about this in discord if this explanation isn't clear enough.

Reply:
I agree flags are not the greatest way to get things done. However, in an attempt to reduce duplicate code and multiple exit points, I couldn't think of another way.

Regarding both the above reviews, since the `success` flag is set to `false` at the beginning, there is no need to display to check if anything other than `true` or `false` was entered, nor is there a need to display an error when a number is not entered.

Reply:
I've removed the flag and made the error messages more descriptive.